setup.py

```diff
@@ -45,12 +39,11 @@
     license="Apache2",
     url=RESOLVER_URL,
     packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
-    py_modules=[RESOLVER_MODULE_NAME],
+    py_modules=["resolver.to_json", "resolver.from_json"],
     entry_points={
         "sceptre.resolvers": [
-            "{}={}:{}".format(
-                RESOLVER_COMMAND_NAME, RESOLVER_MODULE_NAME, RESOLVER_CLASS
-            )
+            "from_json=resolver.from_json:FromJsonResolver",
+            "to_json=resolver.to_json:ToJsonResolver",
```

Comment:
looks good but pulling in PR https://github.com/Sceptre/sceptre-json-resolver/pull/1 will make this entire setup much simpler.  i recommend doing that before merging this PR.